### ISSUE TITLE
Confusing input in predictor.py and predictor-dynamic.py

### ISSUE THREAD

**Issue by jxst539246** [SLP-team/SLP-Core#4]
I found that you used test_data as train_input in your predictor.py and predictor-dynamic.py
`    with tf.name_scope("Train"):
      train_input = PTBInput(config=eval_config, data=test_data, name="TrainInput")
      with tf.variable_scope("Model", reuse=None, initializer=initializer):
        m = PTBModel(is_training=True, config=eval_config, input_=train_input)
      tf.summary.scalar("Training Loss", m.cost)
      tf.summary.scalar("Learning Rate", m.lr)
`
Is there something wrong?

**Comment by VHellendoorn** [comment]
Thanks for your comment. Since I tested both static and dynamic (which learn at test time) versions of the model, I found it easiest for the tester and predictor files to set the test data as a train input and instead disable or enable the learning at test time by setting the learning rate to 0 or 0.1 (or 0.05) respectively. That has the same effect as static vs. dynamic.

Having said that, the TF code is almost identical to Tensorflow's own ptb_word_lm example (as it was at the time) and I encourage you to refer to their newer code instead if you want to use deep learners for code, especially since the API and performance has evolved a lot since. I only included the TF code in the replication folder for reference of what I used at the time; it's not particularly fast or intuitive, as you noticed too.
